vinyl/functions.php

```diff
@@ -49,3 +49,45 @@ function displayRecords(array $records): string {
     }
 } 
 
+/**
+ * Validate input from addform page 
+ *
+ * @param array $input
+ * @return bool False if the fields are left empty. 
+ */
+function validatePost(array $input) {
+    if((!isset($input['artist-name']) || $input['artist-name'] === '')
+    || (!isset($input['record-name']) || $input['record-name'] === '')
+    || (!isset($input['record-size']) || $input['record-size'] === '')) 
+        { return false;
+    } return true; 
+}
+
+/**
+ * if statement, redirect from the form page. 
+ *
+ * @param array $input
+ * @return void sends to different locations
+ */
+function redirect($input) {
+    if ($input === false){
```

Comment:
You can simplify the condition in this if statement like so:

`if (!$input)`

I will have the same effect, by using the ! you're essentially saying the if statement will run if $input is false